pandas/core/generic.py

```diff
@@ -6283,6 +6258,38 @@ def logical_func(self, axis=None, bool_only=None, skipna=None, level=None,
     return set_function_name(logical_func, name, cls)
 
 
+def _ipython_display_(self):
+    # Having _ipython_display_ defined messes with the return value
+    # from cells, so the Out[x] dictionary breaks.
+    # Currently table schema is the only thing using it, so we'll
+    # monkey patch `_ipython_display_` onto NDFrame when config option
+    # is set
+    # see https://github.com/pandas-dev/pandas/issues/16168
+    try:
+        from IPython.display import display
+    except ImportError:
+        return None
+
+    # Series doesn't define _repr_html_ or _repr_latex_
+    latex = self._repr_latex_() if hasattr(self, '_repr_latex_') else None
```

Comment:
maybe we should define his for Series (trivial version) to avoid this kind of logic